packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -6,17 +6,39 @@ import { useCartContext } from '@magento/peregrine/lib/context/cart';
 export const useProduct = props => {
     const {
         item,
-        removeItemMutation,
+        mutations: { removeItemMutation, updateItemQuantityMutation },
```

Comment:
I recall a guideline opposing nested destructuring, but can't find it. I'm definitely guilty of doing this with Apollo hooks, so if that is the case a lot of instances have slipped through. Just calling it out, might be a tech debt thing we should fix, or even better, add a lint rule for it if this is something we do prefer.

Reply:
Let's leave this for now until we can enforce a linting rule for it.